### PR TITLE
Portfolio Project: Popup

### PR DESCRIPTION
**Pull request by samlorlah**
**Portfolio: details popup window**

- Modal Added to HTML
- Modal Styled for Mobile and Desktop
- Event Listener to Close Modal Added
- Event Listener to open Modal Added
- Multiple Modal can be used with different content and it will work dynamically with the right data-target declared

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -222,7 +222,83 @@ <h3 class="work-description">
...
+                            <div class="year">
+                                <h3>2015</h3>
+                            </div>
+                        </div>
+                    </div>
+                    <img src="./asset/images/close-icon-dark.png" height="14" width="14" class="modal-close"
+                        alt="Close Modal Icon">
+                </div>
+                <div class="modal-body">
+                    <img class="desktop-img" src="./asset/images/Modal-Image-Desktop.png" alt="Modal Image">
```
Comment (Munsa1) [inline index.html:253]: - [ ] Kindly fix the modal image in the desktop version as it currently looks strectched when compared to the figma design.

Your design
image

Provided design
image

File: index.html [hunk 2]
```diff
@@ -222,7 +222,83 @@ <h3 class="work-description">
...
+                    <img class="mobile-img" src="./asset/images/Modal-Image-Mobile.png" alt="Modal Image">
```
Comment (Munsa1) [inline index.html:254]: - [ ] Kindly fix the overflow on the modal pop up  for the mobile version. You could add `overflow: hidden` on the x-axis or change the height of the element causing this.

image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @samlorlah @kendoriddy, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (samlorlah) [reply index.html:254]: There is no overflow in the code even after using various mobile screen sizes. However, I have set overflow-x to hidden. Thanks for your review

image

File: index.html (hunk 1 above)
Comment (samlorlah) [reply index.html:253]: Image height corrected. Thanks

File: asset/css/style.css [hunk 3]
```diff
@@ -810,6 +919,90 @@ body {
...
+    display: flex;
+    padding: 60px 0;
+  }
+
+  .modal-description .modal-buttons {
+    margin: 30px 0;
+    padding: 30px 0;
+    border-top: 1px solid #ebecf0;
+    width: 100%;
+  }
```
Comment (1GORDON) [inline asset/css/style.css:1002]: Great job team 🤝

- [ ] [Optional] You may kindly fix these buttons which seem to appear broken as the screen size reduces on the desktop popup.
buttons popup

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @samlorlah & @kendoriddy ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Descriptive Pull request ✔️
- Professional Readme ✔️
- Linters are passing ✔️

## Optional suggestions

- [ ] [Optional] You may give your header a non-transparent like a slightly darker background so that words don't appear underneath as you scroll.
- [ ] [Optional] You should have also stored the data that populates the popup in an Array of objects in a JS file as specified in the project requirements.

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**